deps/GraphBLAS/CMakeLists.txt

```diff
@@ -60,20 +60,20 @@ include_directories ( Source/Template Source Source/Generated Include Demo/Inclu
 if ("${CMAKE_C_COMPILER_ID}" STREQUAL "GNU")
     # cmake 2.8 workaround: gcc needs to be told to do ANSI C11.
     # cmake 3.0 doesn't have this problem.
-    set (CMAKE_C_FLAGS  "-std=c11 -lm -fPIC")
+    set (CMAKE_C_FLAGS  "-g -std=c11 -lm -fPIC")
```

Comment:
Necessary to get GraphBLAS line numbers in stack traces - this can be reverted later, though I don't really have any objection to leaving them on.